.gitignore

```diff
@@ -3,6 +3,8 @@ Gemfile.lock
 .DS_store
 .idea
 .jekyll-metadata
+package.json
```

Comment:
@vogievetsky can you please comment on whether these belong in .gitignore or not?

Reply:
definitely not!